code/modules/mob/mob_defines.dm

```diff
@@ -84,14 +84,6 @@
 	/// Tick time the mob can next move
 	var/next_move = null
 
-	/**
-	  * Magic var that stops you moving and interacting with anything
-	  *
-	  * Set when you're being turned into something else and also used in a bunch of places
-	  * it probably shouldn't really be
-	  */
-	var/notransform = null //Carbon
```

Comment:
the boolean variable and the useless `//Carbon` comment (especially since dead mobs, silicons, and simple animals use it) is what pissed me off so bad to audit it all and change it